device/broadcom/x86_64-broadcom_common/x86_64-broadcom_b27/broadcom-sonic-td3.config.bcm

```diff
@@ -0,0 +1,14 @@
+mem_cache_enable=1
```

Comment:
@geans-pin b27 is TD3, b37 is TD3, are they same?

Reply:
No, one is TD3 X2, one is TD3 X3